ERCS/erc-7989.md

```diff
@@ -0,0 +1,276 @@
+---
+eip: 7989
+title: Ethereum Intent URI (EIURI)
+description: A Universal action URI format for Ethereum (represent and trigger Ethereum JSON-RPC requests)
```

Comment:
You shouldn't need to mention Ethereum in the title or description. It's implied.